src/core_modules/capture-core/components/DataEntry/withSaveHandler/withSaveHandler.js

```diff
@@ -184,18 +185,18 @@ const getSaveHandler = (
         }
 
         handleSaveDialog = () => {
-            this.handleSave(this.state.saveType);
+            this.handleSave(addEventSaveTypes.SAVE_WITHOUT_COMPLETING);
             this.setState({ messagesDialogOpen: false });
         }
 
         handleSave = (saveType?: ?string) => {
-            const { onSave, itemId, id, calculatedFoundation } = this.props;
-            onSave(
-                itemId,
-                id,
-                calculatedFoundation,
-                saveType,
-            );
+            console.log('handleSave', saveType);
```

Comment:
is there any reason why we should keep this console?

Reply:
Good catch! I forgot to remove it. I updated the code. Thank you